fullscreen/api/document-exit-fullscreen-active-document.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<title>Document#exitFullscreen() when the document is not the active document</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<iframe allowfullscreen></iframe>
+<script>
+var t = async_test();
+
+onload = t.step_func(() => {
+  var iframe = document.querySelector("iframe");
+  var documentBeforeNav = iframe.contentDocument;
+
+  iframe.onload = t.step_func(() => {
+    var p = documentBeforeNav.exitFullscreen();
+    assert_true(p instanceof Promise, 'exitFullscreen() returns promise');
```

Comment:
The message is shown when the assert fails, and this message says the opposite of what happened when it fails. Maybe 'exitFullscreen() should return a promise'